cli/test/test-frame-fit-viewer.mjs

```
@@ -0,0 +1,139 @@
+//   1. the frame fills the terminal exactly (no dead space): rows === height;
+//   2. it shows as many lines as fit (not fewer).
+import { EventEmitter } from 'node:events'
+import { render as inkRender } from 'ink'
+import React from 'react'
+import { stripAnsi } from '../src/build/onboarding/ai-fit.ts'
+import { FullscreenAiViewer } from '../src/build/onboarding/ui/components.tsx'
+
+let passed = 0
+let failed = 0
+function test(name, fn) {
+  try { fn(); passed++; console.log(`✔ ${name}`) }
```

ESLint

[error] This line has 4 statements. Maximum allowed is 1.

(style/max-statements-per-line)